src/Surfnet/StepupSelfService/SelfServiceBundle/Form/Type/RemoteVetValidationType.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/**
+ * Copyright 2014 SURFnet bv
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\StepupSelfService\SelfServiceBundle\Form\Type;
+
+use Surfnet\StepupSelfService\SelfServiceBundle\Command\RemoteVetValidationCommand;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+use Symfony\Component\Form\Extension\Core\Type\CollectionType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class RemoteVetValidationType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder->add('matches', CollectionType::class, [
+            // each entry in the array will be an "email" field
+            'entry_type' => RemoteVetAssertionMatchType::class,
+            // these options are passed to each "email" type
+            'entry_options' => [
+                'attr' => ['class' => 'assertion_match'],
+            ],
+        ]);
+
+        $builder->add('valid', CheckboxType::class, [
```

Comment:
The purpose of this checkbox remains unclear to me. Making it required => true might not be intended?

Reply:
This is the ultimate manual RV matching check, only if this is checked the token could be vetted.